docs/components/servo/pi.md

```diff
@@ -86,11 +86,11 @@ The following attributes are available for `pi` servos:
 | ---- | ---- | --------- | ----------- |
 | `pin` | string | **Required** | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of the pin the servo's control wire is wired to on the [board](/components/board/). |
 | `board` | string | **Required** | `name` of the [board](/components/board/) the servo is wired to. |
-| `min` | float | Optional | The minimum angle in degrees that the servo can reach. <br> Default = `0.0` <br> Range = [`0.0`, `180.0`] |
-| `max` | float | Optional | The maximum angle in degrees that the servo can reach. <br> Default = `180.0` <br> Range = [`0.0`, `180.0`] |
+| `min` | float | Optional | The minimum angle in degrees you want your servo to reach while rotating. <br> Default = `0.0` <br> Range = [`0.0`, `180.0`] |
```

Comment:
I think describing it as a software limit will help users understand. What do you think of something like
```suggestion
| `min` | float | Optional | Sets a software limit on the minimum angle in degrees your servo can rotate to. <br> Default = `0.0` <br> Range = [`0.0`, `180.0`] |
```

Reply:
great suggestion, sounds good to me!